eos-reclaim-swap

```diff
@@ -29,7 +29,8 @@ class SwapRemover():
         disk = None
 
         # /dev/sd[a-z][0-9]
-        if p.startswith('/dev/sd'):
+        # /dev/vd[a-z][0-9]
+        if p.startswith('/dev/sd') or p.startswith('/dev/vd'):
```

Comment:
This is a bit belt-and-braces because since the repartition script has never worked on `/dev/vd*`, we can never have created swap there. But maybe simpler to just handle this case than to explain it.